integrations/deploy.yaml

```diff
@@ -214,7 +214,7 @@
 - id: deploy-docker
   meta:
     name: Docker
-    link: ''
+    link: 'https://www.apple.com/macos/ventura/'
```

Comment:
wrong url?

Reply:
Should it be https://hub.docker.com/r/netdata/netdata?

Reply:
Fixed.